taxonomies/ingredients.txt

```diff
@@ -18188,6 +18188,10 @@ zh:亞麻子油
 wikidata:en:Q818405
 from_palm_oil:en:no
 
+<en:linseed
+en:linseed-seeds
```

Comment:
the entries en: es: etc. should not have dashes, it should be "linseed seeds", "semillas de linaza", with spaces and accents

Reply:
ok